docs/installing.md

```diff
@@ -0,0 +1,18 @@
+# Installing Rancher Desktop
+
+Rancher Desktop is delivered as a desktop application. You can download it from
+the [releases page on GitHub](https://github.com/rancher-sandbox/rd/releases).
+
+When run for the first time or when changing versions, Kubernetes container
+images are downloaded. It may take a little time to load on first run for a new
+Kubernetes version.
+
+Note, the Windows installer is not yet signed. This will be coming soon. In the
```

Comment:
I'd say "This will happen soon." but it's a matter of preference

Reply:
Updated